### PR TITLE
normalization faulty

### PR DESCRIPTION
**Pull request by tedchou12**
normalization faulty

### CODE REVIEW

**Comment by tedchou12** [comment]
https://zaffnet.github.io/batch-normalization
https://towardsdatascience.com/implementing-batch-normalization-in-python-a044b0369567

**Comment by tedchou12** [comment]
https://colah.github.io/posts/2015-08-Backprop/

**Comment by tedchou12** [comment]
https://kratzert.github.io/2016/02/12/understanding-the-gradient-flow-through-the-batch-normalization-layer.html
